src/components/Navbar/Navbar.js

```diff
@@ -51,19 +91,36 @@ class NavbarUnlogged extends Component {
                     className="form-check-input"
                     name="optionsRadios"
                     id="optionsRadios2"
-                    value="option2"
+                    value="employees"
+                    onChange={event => this.setState({ option: event.target.value })}
                   />
                   Pracobiorcy
                 </label>
               </div>
             </fieldset>
-            <button className="btn btn-secondary mx-3 my-2 my-sm-2" type="submit">
-              Szukaj
-            </button>
+            <Link
+              className="nav-link"
+              to={{
+                pathname: routeToSearchPage
+              }}
+            >
+              <button
+                className="btn btn-secondary mx-3 my-2 my-sm-2"
+                type="submit"
+                onClick={() =>
+                  this.props.actions.searchOffers(
+                    this.state.phrase,
+                    this.state.option
+                  )
+                }
+              >
+                Szukaj
+              </button>
+            </Link>
           </form>
           <button className="btn btn-secondary" type="button">
             Stwórz ofertę
-            </button>
+          </button>
           <ul className="navbar-nav ml-auto">
             <li className="nav-item dropdown">
               <a
```

Comment:
We should address it in separate task, but using \<a\> tags will cause DOM to rerender - we don't want this (consider this just a suggestion, can be omitted for now)

Reply:
Well that would resolve problem that i get on changing pages in pagination. That will be ommited for now though.